### ISSUE TITLE
Defining "Reproducible Science"

### ISSUE THREAD

**Issue by burkesquires** [Reproducible-Science-Curriculum/RR-Jupyter-Hackathon-Jan-2016#7]
In thinking about reproducible science, do we want to clarify that we are in fact talking about "reproducible data analysis"? Perhaps this is obvious as well. 

Perhaps it would help to clearly define where are starting point(s) and end point(s) are, which may simply bring us back to our learning objectives! :-)

**Comment by hlapp** [comment]
Yes, reproducible computational research. Not bench research 😄 Also, I like to remind myself when thinking about definitions that what we are aiming for to teach with the curriculum is not how to conduct "reproducible science" (as then one would need to define what constitutes reproducible science and what does not), but to teach practices that help researchers to conduct their science _more reproducibly_.

**Comment by burkesquires** [comment]
Love the idea of conducting science _more reproducibly_!. Here is an example of steps to reproducible [research](http://kbroman.org/steps2rr/)
